docs/manual/java/concepts/Immutable.md

```diff
@@ -57,7 +57,39 @@ In sbt:
 
 @[lagom-immutables](code/lagom-immutables.sbt)
 
-Immutables integrates with popular IDEs. Follow the instructions for [[Eclipse|ImmutablesInIDEs#Eclipse]] or [[IntelliJ IDEA|ImmutablesInIDEs#IntelliJ_IDEA]] to add the Immutables annotation processor to your IDE.
+Immutables integrates with popular IDEs. Follow the instructions for [[Eclipse|ImmutablesInIDEs#eclipse]] or [[IntelliJ IDEA|ImmutablesInIDEs#intellij-idea]] to add the Immutables annotation processor to your IDE. We've found the integration with IntelliJ IDEA a bit cumbersome though.
+
+### Lombok
+
+You can achieve the same simplicity with Lombok. Here's yet another example of a `User` implemented with Lombok:
+
+@[lombok-immutable](code/docs/home/immutable/LombokUser.java)
+
+That will create:
+
+ * getters for each field
+ * correct `equals`, `hashCode` and a human-friendly `toString`
+ * a constructor requiring all fields.
+
+This example code is not demonstrating other Lombok feature like `@Builder` or `@Wither` which will help you create builder and copy methods.
+
+Lombok is a simple dependency:
+
+```xml
+<dependency>
+    <groupId>org.projectlombok</groupId>
+    <artifactId>lombok</artifactId>
+    <version>1.16.12</version>
+</dependency>
+```
+
+or in sbt:
+
+@[lagom-immutables-lombok](code/lagom-immutables.sbt)
+
+
+Lombok integrates with popular IDE's too. In order to use Lombok in IntelliJ IDEA you'll need the [Lombok Plugin for IntelliJ IDEA](https://plugins.jetbrains.com/idea/plugin/6317-lombok-plugin) and you'll also need to enable Annotation Processing (`Settings / Build,Execution,Deployment / Compiler / Annotation Processors` and tick `Enable annotation processing`). Using Lombok in Eclipse requires running `java -jar lombok.jar` (see the video at [Project Lombok](https://projectlombok.org/)).
+
```

Comment:
If we think Lombok is a better alternative, we should put it before Immutables in the documentation.